packages/core/src/types/index.ts

```diff
@@ -178,7 +178,8 @@ export interface TypographyProps {
     | "heading4"
     | "heading5"
     | "heading6"
-    | "span";
+    | "span"
+    | string;
```

Comment:
why the addition of `string` here?

Reply:
turns out I didn't need that after I typed out the react state... removed